build/index.js

```diff
@@ -245,12 +245,12 @@ async function buildLiveSamplePageFromURL(url) {
         sampleIDObject
       );
       if (liveSamplePage.flaw) {
-        return liveSamplePage.flaw.errorMessage;
+        throw liveSamplePage.flaw;
```

Comment:
it's best practice to only throw error objects (even though JS unfortunately allows you to throw _anything_), reasons being: catching code tends to expect an error object and only with an error object do we get a stack trace